code/style.css

```diff
@@ -1,58 +1,264 @@
+  font-family: 'Open Sans', sans-serif;
+  font-size: 23px;
 }
 
 p {
-  margin: 1px;
+  margin: 10px;
   font-weight: 300;
-  background-color: #fff;
+  border-radius: 10px;
+  padding: 10px;
+  font-size: 20px;
+  color: #000000;
+  font-family: 'Open Sans', sans-serif;
+}
+
+.main-wrapper {
+  padding: 20px;
+  background-image: url("/code/assets/diamond-background.png");
+  background-size: 150px;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.header-container {
+  margin-top: 30px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: #9dccc9;
+  padding: 20px;
+}
+
+.header-container h2 {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 100;
+  font-size: 28px;
+  padding-top: 10px;
+  margin: 0px;
+}
+
+.header-title {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-bottom: 1px solid #000000;
+  padding: 0px 30px;
+}
+
+.header-title h1 {
+  font-size: 50px;
+  margin-bottom: 10px;
+  margin-top: 5px;
+}
+
+.header-title img {
+  border-radius:45%;
+  padding: 5px;  
 }
 
-.cards {
-  margin: 50px auto;
+.cards {  
+  background: #cccccc;
+  margin: 30px auto;
   width: 90%;
-  height: 600px;
-  background: #ccc;
+  /* height: 900px; */
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center; 
+}
+
+.card-container {
+  padding: 20px;
+  width: 100%;
   display: flex;
   justify-content: space-around;
   align-items: center;
+  text-align: center;
+}
+
+.card-header-container {
+  padding: 10px;
+}
+
+.card-header-container h2 {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 100;
 }
 
 .killer-deck {
-  width: 250px;
-  height: 400px;
-  background: #000;
+  width: 500px;
+  height: 550px;
   text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+  margin-right: 30px;
+  border-radius: 5px;
+  background-image: url("./assets/diamond-background-card.png");
+  background-size: 20px;
+  cursor: pointer;
+  box-shadow: 0 3px 5px 0 #555252;
+}
+
+.killer-deck p {
+  margin: 0px;
 }
 
 .weapon-deck {
-  width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
-  text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+  width: 500px;
+  height: 550px;
+  margin-right: 30px;
+  border-radius: 5px;
+  background-image: url("./assets/diamond-background-card.png");
+  background-size: 20px;
+  background-repeat: repeat;
+  cursor: pointer;
+  box-shadow: 0 3px 5px 0 #555252;
 }
 
-.room-deck {
-  width: 250px;
-  height: 400px;
-  background: midnightblue;
+.weapon-deck p {
+  margin: 0px;
+}
 
+.room-deck {
+  width: 500px;
+  height: 550px;
   text-align: center;
-  padding-top: 30px;
-  border-radius: 16px;
+  border-radius: 5px;
+  background-image: url("./assets/diamond-background-card.png");
+  background-size: 20px;
+  cursor: pointer;
+  box-shadow: 0 3px 5px 0 #555252;
+}
+
+.room-deck p {
+  margin: 0px;
+}
```

Comment:
Re: The above example I mentioned with extracting out shared properties – this is a perfect example, given the three classes room-deck, killer-deck and weapon-deck have a lot of identical properties. 

What I would do here is to:
i) take all the duplicate properties of these x-deck classes, 
ii) put them in a separate class (and call it maybe .deck), and 
iii) assign the HTML card article elements the additional class "deck", like follows:

```
.deck {
  width: 500px;
  height: 550px;
  border-radius: 5px;
  cursor: pointer;
  box-shadow: 0 3px 5px 0 #555252;
  background-image: url("./assets/diamond-background-card.png");
  background-size: 20px;
}

.deck p {
  margin: 0px;
}

```
...and in the HTML file, you could structure the elements as such:

```
<article class="deck killer-deck" id="killerCard">...</article>
<article class="deck weapon-deck" id="weaponCard">...</article>
<article class="deck room-deck" id="roomCard">...</article>
```
Adding multiple classes to a HTML element by separating them with a blank space is (in my opinion) a great way to get some peace of mind. But yeah, totally up to you :)

Reply:
Very true Axel! I will implement this idea for my future projects as it makes more sense to have one class name for elements that are being styled the same way 👍

Reply:
@AxelHammarback I decided to implement this as I thought it would make my code cleaner! Thanks for the pro-tip.